test/unit/test_utils.py

```diff
@@ -139,49 +138,43 @@ def setUp(self):
         )
         self.mock_aws_encryption_sdk_instance.decrypt.return_value = VALUES["data_key"]
         self.mock_aws_encryption_sdk_instance.encrypt.return_value = VALUES["encrypted_data_key"]
-
-    def tearDown(self):
```

Comment:
Missing a `yield`.

Reply:
I wrote it in the commit message but didn't add it to the code, my bad. I already added it in the last commit.

Reply:
Yup, saw that commit come through right as I published that comment. 👍